### PR TITLE
Kam daemon2

### PR DESCRIPTION
**Pull request by kameyer226**
Removed hyphen as it lead to bad Python syntax error. Replaced "my-daemon" with "aerotech" as my-daemon is a cookiecutter term.

### CODE REVIEW

**Comment by kameyer226** [comment]
This has been re-submitted for examination.

**Comment by ksunden** [comment]
This is still removing 11 files, and adding nothing... what does `git status` show?

**Comment by kameyer226** [comment]
Fixed,  vscode "add" and "untracked" similar.   Hyphen directory causes syntax issue, "my-daemon" replaced

**Review by kameyer226 — COMMENTED** [review]
merge appears in line with intended changes for this rev.

File: aerotech_101smc2/aerotech_101smc2/_aerotech.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 class Aerotech(ContinuousHardware):
-    _kind = "my-daemon"
+    _kind = "Aerotech_101smc2"
```
Comment (ksunden) [inline aerotech_101smc2/aerotech_101smc2/_aerotech.py:10]:
```suggestion
    _kind = "aerotech-101smc2"
```

the standard here is all lower case, dashes between words

**Review by ksunden — COMMENTED** [review]
Should edit the `setup.py` line which references `my-daemon` with the updated file name and replace `yaqd-my-daemon` with `yaqd-aerotech-101smc2`

**Pull request merged**